common/flatpak-dir.c

```diff
@@ -3098,6 +3100,12 @@ flatpak_dir_do_resolve_p2p_refs (FlatpakDir             *self,
           else
             resolve->resolved_metadata = g_bytes_new (xa_metadata, strlen (xa_metadata) + 1);
 
+          if (g_variant_lookup (commit_metadata, "xa.download-size", "t", &installed_size))
```

Comment:
Looks like the download size is being put in the installed size variable and vice versa